src/Provisioning.ts

```diff
@@ -87,6 +87,20 @@ export class Provisioner {
         return (currentCount >= this.main.config.provisioning?.limits?.room_count);
     }
 
+    private async determineSlackIdForRequest(matrixUserId, teamId) {
+        const matrixUser = await this.main.datastore.getMatrixUser(matrixUserId);
+        if (matrixUser === null) {
+            throw Error('No users found');
```

Comment:
Nit-pick: User**s**? We're just looking for one user.

Reply:
We should also be returning null.